spec/chromium-spec.js

```diff
@@ -998,52 +998,58 @@ describe('chromium feature', () => {
   })
 
   describe('PDF Viewer', () => {
-    const pdfSource = url.format({
-      pathname: path.join(fixtures, 'assets', 'cat.pdf').replace(/\\/g, '/'),
-      protocol: 'file',
-      slashes: true
-    })
-    const pdfSourceWithParams = url.format({
-      pathname: path.join(fixtures, 'assets', 'cat.pdf').replace(/\\/g, '/'),
-      query: {
-        a: 1,
-        b: 2
-      },
-      protocol: 'file',
-      slashes: true
-    })
-
-    function createBrowserWindow ({plugins, preload}) {
-      w = new BrowserWindow({
-        show: false,
-        webPreferences: {
-          preload: path.join(fixtures, 'module', preload),
-          plugins: plugins
-        }
-      })
-    }
+    before(function () {
+      if (!parseInt(process.env.PDF_VIEWER_ENABLED)) {
+        return this.skip()
+      }
 
-    function testPDFIsLoadedInSubFrame (page, preloadFile, done) {
-      const pagePath = url.format({
-        pathname: path.join(fixtures, 'pages', page).replace(/\\/g, '/'),
+      const pdfSource = url.format({
```

Comment:
Define it as `this.pdfSource  = ...` and then refer to it as `this.pdfSource` from the tests (`it(...)`).